src/components/Room.styled.tsx

```diff
@@ -0,0 +1,13 @@
+import styled, { createGlobalStyle } from "styled-components";
+
+export const StyledMain = styled.main`
```

Comment:
```suggestion
export const StyledMain = styled.div`
```

`<main>` is a semantic HTML element for the dominant content on the page. For example the main paragraph in a blog/article. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main

In this case, you are wrapping the whole room component in the `<main>` element which would be semantically incorrect. 

I think a `<div>` component is more apt here.

Reply:
You're right. I fixed it, div is more appropriate.